packages/documentation/copy/en/handbook-v2/Object Types.md

```diff
@@ -579,7 +579,7 @@ draw({ color: "blue", radius: 42 });
 draw({ color: "red", raidus: 42 });
 ```
 
-## Interfaces vs. Intersections
+## Extensions vs. Intersections
```

Comment:
If we're going to change this, it should be to "Interface extension vs intersections".

Reply:
What about "Interface extension vs. Type alias intersection"? Would that be more appropriate, or too verbose?

Reply:
I wouldn't say "type alias"; you can intersect anything. So I would just use the wording I suggested.